### PR TITLE
include latest changes to edge analysis in CMGTools

### PR DESCRIPTION
**Pull request by mdunser**
includes first version of summer student code for DPS

### CODE REVIEW

File: TTHAnalysis/macros/prepareEventVariablesFriendTreeEdge.py [hunk 1]
```diff
@@ -0,0 +1,276 @@
+#!/usr/bin/env python
```
Comment (gpetruc) [inline TTHAnalysis/macros/prepareEventVariablesFriendTreeEdge.py:1]: There shouldn't be the need to duplicate almost the entirety of this file.

Can we discuss how to keep only one, and refactor away in other python files the specific parts of edge and multilepton?

Comment (gpetruc) [inline TTHAnalysis/macros/prepareEventVariablesFriendTreeEdge.py:1]: @peruzzim

**Comment by gpetruc** [comment]
There is a merge conflict in this PR, in samples_13TeV_RunIISpring16MiniAODv2.py for the `WWDouble` process. 
The two conflicting updates also have slightly different cross sections: 1.61704 in the current head, 1.64 in your PR.
Can you merge the cmg-central HEAD in your branch, fix the conflict, pick the right cross section, and push again to this branch to update the PR?
Thanks
Giovanni

**Comment by mdunser** [comment]
I hope that fixed it?! If not, let me know.
I fixed the cross section to what is on the twiki (https://twiki.cern.ch/twiki/bin/viewauth/CMS/SummaryTable1G25ns).
It’s a < 2% difference anyway…
-m

On 18 Jun 2016, at 11:19, Giovanni Petrucciani <notifications@github.com<mailto:notifications@github.com>> wrote:

There is a merge conflict in this PR, in samples_13TeV_RunIISpring16MiniAODv2.py for the WWDouble process.
The two conflicting updates also have slightly different cross sections: 1.61704 in the current head, 1.64 in your PR.
Can you merge the cmg-central HEAD in your branch, fix the conflict, pick the right cross section, and push again to this branch to update the PR?
Thanks
Giovanni

**Pull request merged**